precedence.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+    "use strict";
+
+    function Precedence() {
+        this.numbers = [];
+        this.operations = [];
+        this.turnOn = false;
+        this.finalResult = 0;
+    }
+
+    Precedence.prototype.returnValue = function() {
```

Comment:
*return* is implicit for every method, so do not use it in method names. A good name for here would be *calculateResult()* or similar.